### PR TITLE
Pin module

### PR DESCRIPTION
**Pull request by rachelcosterhaventec**
The pull request is for the story f155574056

In addition to the story, I have also changed the name of the repository and the project files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# Angular2 Pin Component
-
-A modular pin component made in angular2.
```
Comment (justincrosbiehaventec) [inline README.md]: I think the README needs a bit of work - needs to mention npm install, and assume the end user has no idea how to use it

**Comment by justincrosbiehaventec** [comment]
If you type the wrong pin its cumbersome to undo - should focus clear an input?
Can you make the README more user-friendly?
Has it been tested with a keypad with numlock on? (sends different keycodes)
Has it been tested inside an iOS and Android device?

File: README.md (hunk 1 above)
Comment (rachelcosterhaventec) [reply README.md]: I haven't set up the npm install as yet so I don't really want to include that until it's on npm

**Comment by justincrosbiehaventec** [comment]
The functionality seems good.
In its current form, it can't be published to the npm registry, so need to be able to build it to that form
Also is missing unit tests
Has it been tested across all required platforms? I would suggest building it into a mobile app (e.g. Ionic) as an npm dependency and deploying it onto Android and iOS devices.

**Pull request merged**